openassessment/workflow/test/test_models.py

```diff
@@ -0,0 +1,166 @@
+""" Tests for ORA workflow models """
+
+from contextlib import contextmanager
+import ddt
+import mock
+from freezegun import freeze_time
+
+from django.utils.timezone import now
+
+from openassessment.test_utils import CacheResetTest
+from openassessment.workflow.errors import AssessmentWorkflowInternalError
+from openassessment.workflow.models import TeamAssessmentWorkflow, AssessmentWorkflowStep
+from openassessment.workflow.test.factories import AssessmentWorkflowStepFactory
+
+
+@ddt.ddt
+class TeamAssessmentTest(CacheResetTest):
+    """ Tests for the TeamAssessmentWorkflow model """
+    team_submission_uuid = 'team-submission-uuid'
+    submission_uuids = ['submission-uuid-' + str(i) for i in range(5)]
+    course_id = 'edx/Teamwork/Cooperation'
+    item_id = 'this-is-an-item-id-i-guess'
+    MOCK_TEAM_SUBMISSION = {
+        'team_submission_uuid': team_submission_uuid,
+        'submission_uuids': submission_uuids,
+        'course_id': course_id,
+        'item_id': item_id
+    }
+
+    def setUp(self):
+        super(TeamAssessmentTest, self).setUp()
```

Comment:
in Python 3+ no need to pass parameters when calling super. this can be called:
`super().setUp()`